src/ui/components/NetworkMonitor/RequestTable.tsx

```diff
@@ -51,6 +53,17 @@ const RequestTable = ({
         className={classNames("flex h-full w-full flex-col overflow-x-auto", styles.request)}
         {...getTableProps()}
       >
+        {isCopied || shouldShowLoading ? (
+          <div className="absolute z-50 grid h-56 grid-cols-1 content-end place-self-center">
+            <div
+              id="showCopied"
+              className={`mb-1.5 flex shrink rounded-lg bg-black p-1.5 text-center text-white opacity-100 shadow-2xl transition-all duration-700 ease-in-out`}
+            >
+              {`${shouldShowLoading ? "Copying" : "Copied"} to clipboard`}
```

Comment:
1. This was not due to `shouldShowLoading` flag, but due to `setTimeout` I have used to defer setting the state by `300ms`. I did this so when the tooltip changes from `Copying to clipboard` to `Copied to clipboard the tooltip is not shown for few milliseconds and that gives a clear visual for the user instead of just text changing from `Copying` to `Copied` . Here `Copying` to `Copied` happens only for the case where there is a request body, for all other cases it just displays `Copied` in the first place and yeah that delay with `setTimeout` has caused issue with items where there is no body. 

2. I was not sure how it is generally shown across the application, so I decided to have it centred.